### ISSUE TITLE
sliding menu is stuck while open and close.

### ISSUE THREAD

**Issue by kiranboghra** [jfeinstein10/SlidingMenu#599]
I have use this nice library in my project but is stuck while opening and closing menu So please help if any one face this issue and solved. And when I only open and close menu without refresh fragment view its work perfact and not stuck.

**Comment by mjkohoh** [comment]
did you open the hardwareAccelerated

**Comment by kiranboghra** [comment]
Yes I have open the hardwareAccelerated but it stucks. My problem is when I close menu and refresh Fragment than only stuck if I close menu without refresh Fragment than is is not stuck .

**Comment by marcelotadeujr** [comment]
@kiranboghra i have same trouble and cannot solve then...

Any suggestion?

Thanks.

**Comment by Kilnn** [comment]
@kiranboghra I have the same problem with you. And i also open the hardwareAccelerated,but it only work on smooth scroll . when i call the method showMenu(),  there is a white line flash,and then ,scroll
Have you solved the problem,please tell me,thanks very much

**Comment by Kilnn** [comment]
@kiranboghra  @BM9MobileBrasil  
I add this code in my only activity which extends the SlidingFragmentActivity:
getWindow().getDecorView().setBackgroundResource(android.R.color.transparent);
There is a little effect.
You can have a try

**Comment by Kishanjvaghela** [comment]
Change Fragment while menu drawer is closed, rather than view is sliding.

 sm.setOnClosedListener(new SlidingMenu.OnClosedListener() {

```
        @Override
        public void onClosed() {
            changeFragment(selectedPosition);
        }
    });
```
